libs/libcommon/src/libcommon/constants.py

```diff
@@ -5,3 +5,14 @@
 CACHE_MONGOENGINE_ALIAS = "cache"
 QUEUE_MONGOENGINE_ALIAS = "queue"
 QUEUE_TTL_SECONDS = 604_800  # 7 days
+
+
+PROCESSING_STEP_CONFIG_NAMES_VERSION = "1.0.0"
```

Comment:
Not sure if this is the best way to manage the versions but, I didnt find another way to have them in admin project without importing workers as dependency (I woulndt like to do it since will be having access to extra logic).

Reply:
Good point. You're right, services/admin does not have access to the job runners' versions.

Instead of an admin endpoint, maybe we want an one-time job (like db migration -> cache_refresh?) run before deploying a new version. We would hardcode the current job runner versions there, and new jobs would be created to update the obsolete cache entries. wdyt?

Reply:
Good idea to share the versions in libcommon. We will have to ensure updating it on every change in the job runners' code (as of now, where the version is inside the same file... not sure if it changes anything - CI might help here?).

Reply:
> Instead of an admin endpoint, maybe we want a one-time job (as database migration -> cache_refresh?) run before (or in parallel to) deploying a new version. We would hardcode the current job runner versions in the script, and new jobs would be created to update the obsolete cache entries. wdyt?

Yeah, I also think that is a better Idea but was not sure if that will increase k8s/docker resources, if that is not a problem, lets do it.

Reply:
no, I don't see why the resources would be a problem.